crates/merod/src/main.rs

```diff
@@ -7,16 +7,20 @@ use tracing_subscriber::prelude::*;
 use tracing_subscriber::{registry, EnvFilter};
 
 use crate::cli::RootCommand;
+use crate::version::check_for_update;
 
 mod cli;
 mod defaults;
+mod version;
 
 #[tokio::main]
 async fn main() -> EyreResult<()> {
     setup()?;
 
     let command = RootCommand::parse();
 
+    check_for_update().await;
```

Comment:
`tokio::spawn` this task

```suggestion
    let _ignored = tokio::spawn(async {
        if let Err(err) = check_for_update().await {
            error!(.....);
        }
    });
```